docs/v2-release-notes.md

```diff
@@ -2,6 +2,16 @@
 
 This page summarizes changes for each major update in Timeplus Enterprise v2 and Timeplus Proton, including new features and important bug fixes.
 
+## Jul 22, 2024
+
+### Timeplus Proton v1.5.14
+  * New feature: Remote user-defined functions (UDFs) are now supported.
+    * Example: `CREATE REMOTE FUNCTION ip_lookup(ip string) RETURNS string URL 'https://abc.lambda-url.us-west-2.on.aws/'`
+
```

Comment:
timeplus-native-jdbc v2.0.2 (https://github.com/timeplus-io/timeplus-native-jdbc/releases/tag/v2.0.2)
* support bigint and decimal type 
* Bugfix: invalid version for lowcardinality keys